apps/build-notification/src/githubPrComment.ts

```diff
@@ -0,0 +1,122 @@
+import type { Octokit } from "@octokit/rest";
+
+import type {
+  BuildAggregatedStatus,
+  PullRequest,
+} from "@argos-ci/database/models";
+import { getRedisLock } from "@argos-ci/web";
```

Comment:
`@argos-ci/web` should not be a dependency of `built-notification`. You should create a `@argos-ci/common` or `@argos-ci/util`.